### ISSUE TITLE
Ensure meaningful order of content across spreads and within pages

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epub-specs#2909]
### Section

Ensure meaningful order of content across spreads and within pages

### Describe the problem

The techniques document includes a section entitled "Ensure meaningful order of content across spreads and within pages," but a section with the same title is available in EPUB Accessibility Techniques 1.1: https://www.w3.org/TR/epub-a11y-tech-11/#spread-order and the two sets of guidelines are not consistent.

### Describe the fix or new feature you propose

I suggest referring to EPUB Accessibility Techniques 1.1.

**Comment by clapierre** [comment]
Actually, I disagree here, there are unique issues that Fixed Layout publications have, which are outlined in the "EPUB Accessibility - Fixed Layout Techniques" document section "3.2 Content Access", and the "3.2.1 Ensure meaningful order of content across spreads and within pages". 

What I do think is missing here is that we should refer to the "EPUB Accessibility Techniques 1.1" section "[4.2.1 Ensure meaningful order of content across spreads](https://www.w3.org/TR/epub-a11y-tech-11/#spread-order)" from "3.2.1 Ensure meaningful order of content across spreads and within pages" in the FXL Techniques document.

If @gregoriopellegrino and @wareid agree either I can make this change to the existing PR #2878 or wait for that to be merged and create a new PR. Or feel free to make this change yourself Wendy.

**Comment by gregoriopellegrino** [comment]
Yes, I understand. Fine for me.

**Comment by avneeshsingh** [comment]
copy pasting the discussion from Feb 19 meeting minutes, just for record:
Charles LaPierre These techniques should be in the EPUB accessibility techniques document

Charles L.: Decision: Synchronize names of this heading of EPUB Accessibility and fixed layout techniques, refer to EPUB Accessibility Techniques 1.2, and then decide whether specific guidance is needed for spreads in fixed layout techniques, while remaining technology-independent.
